tools/wait_for_service.sh

```diff
@@ -6,9 +6,21 @@ if [ "$#" -ne 2 ]; then
     exit "Illegal number of parameters"
 fi
 
+set -e
+
 CONTAINER="$1"
 PORT="$2"
-IP=$(/usr/bin/docker inspect -f {{.NetworkSettings.IPAddress}} "$CONTAINER")
+IP=$(docker inspect -f {{.NetworkSettings.IPAddress}} "$CONTAINER")
 echo "$CONTAINER IP is $IP"
 
-tools/wait-for-it.sh -h $IP -p $PORT
+if [ `uname` = "Darwin" ]; then
+  # Direct access to IPs is not supported on Mac
+  # https://docs.docker.com/docker-for-mac/networking/
+  # But we can run wait-for-it from another container
+  docker run --rm -d --name wait-helper ubuntu sleep infinity || echo "We can continue if the wait-helper exists"
```

Comment:
I see that here you used `ubuntu` image, but in other place (https://github.com/esl/MongooseIM/pull/1908/files#diff-4e96dc6d6584239b1e20b34ff2804857R52) you used `busybox`. Is it possible to have only one image?

Reply:
And why do we need docker container for this wait-for-it script? Isn't there other way?

Reply:
because mac suuu...

I mean it has old bash, no timeout command, it's hard to check tcp connection (ok, nc?), and wait_for_service.sh waits for port to appear on private IP (which is not available in Mac).

The ubuntu image is only used for Mac (i.e. once you download the image, it runs fast).

Reply:
You can test performance with

```erlang
DB=cassandra ./tools/travis-setup-db.sh
DB=mysql ./tools/travis-setup-db.sh
...
````

Reply:
All right, can we use only one image? `ubuntu` or `busybox`?

Reply:
Actually it'd be perfect to use `busybox` here and for the uploader. If that's not possible, we can live with this 2 I think.

Reply:
uploader was removed.
busybox does not have bash :D